nixos/modules/services/web-servers/keter/default.nix

```diff
@@ -34,6 +34,9 @@ Keep an old app running and swap the ports when the new one is booted.
           host = "*4";
```

Comment:
L29: `globalKeterConfig` -> `settings` perhaps? And why is it `keterPackage` and `keterRoot` also? This is `services.keter.*`; keter is implied.

Reply:
You'll need to use `mkRenamedOptionModule`.

Reply:
got it, thanks!